server.js

```diff
@@ -204,6 +204,15 @@ const workspaceProxyOptions = {
             return 'not-matched';
         }
     },
+    onError: (err, req, res) => {
+        logger.err(`Error proxying workspace request: ${err}`);
+        try {
+            res.writeHead(500, { 'Content-Type': 'text/plain' });
+            res.send('There was an error proxying this workspace request');
+        } catch (err2) {
```

Comment:
Not that I really mind, but this could be just `err` again, right? I try and stick to the convention that the most recently used callback is always `callback` and the most recent error is `err` (unless there is some other good reason, of course).